askama/src/filters/mod.rs

```diff
@@ -511,6 +511,42 @@ pub fn wordcount(s: impl ToString) -> Result<usize, Infallible> {
     wordcount(s.to_string())
 }
 
+/// Converts a type to a `String`.
+#[inline]
+pub fn string(s: impl ToString) -> Result<String, Infallible> {
+    Ok(s.to_string())
+}
+
+/// Return a title cased version of the value. Words will start with uppercase letters, all
+/// remaining characters are lowercase.
+#[inline]
+pub fn title(s: impl ToString) -> Result<String, Infallible> {
+    let s = s.to_string();
+    let mut need_capitalization = true;
+
+    // Sadly enough, we can't mutate a string when iterating over its chars, likely because it could
+    // change the size of a char, "breaking" the char indices.
+    Ok(s.chars()
```

Comment:
I've been thinking about this but this latest reformatting (due to `new()` -> `with_capacity()`) I think pushes it over the line: a `for` loop would be a nice simplification instead of the `fold()` call.

Reply:
Ok, updating.